E/Elfutils/build_tarballs.jl

```diff
@@ -1,29 +1,46 @@
 # Note that this script can accept some limited command-line arguments, run
 # `julia build_tarballs.jl --help` to see a usage message.
-using BinaryBuilder
+using BinaryBuilder, Pkg
 
 name = "Elfutils"
-version = v"0.177"
+version = v"0.179"
 
 # Collection of sources required to build Elfutils
 sources = [
     ArchiveSource("https://sourceware.org/elfutils/ftp/$(version.major).$(version.minor)/elfutils-$(version.major).$(version.minor).tar.bz2",
-                  "fa489deccbcae7d8c920f60d85906124c1989c591196d90e0fd668e3dc05042e"),
+                  "25a545566cbacaa37ae6222e58f1c48ea4570f53ba991886e2f5ce96e22a23a2"),
+    DirectorySource("./bundled"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir/elfutils-*/
+if [[ ${target} = *-musl* ]] ; then
+    for patchfile in $WORKSPACE/srcdir/patches/*; do
+        atomic_patch -p1 $patchfile
+    done
+    cp $WORKSPACE/srcdir/error.h src/
+    cp $WORKSPACE/srcdir/error.h lib/
+
+    # For some reason, just installing bsd-compat-headers doesn't work
+    apk add bsd-compat-headers
+    mkdir -p $prefix/include/sys
+    # Skip warning macro at top of file
+    tail -n +2 /usr/include/sys/cdefs.h >$prefix/include/sys/cdefs.h
+    autoreconf -vif
```

Comment:
It isn't a good idea to copy the file to `${includedir}`: they'll be stored in the tarball.  At very least you need to delete it at the end of the build.  Alternatively, copy it somewhere else and set `CPPFLAGS=-I...` appropriately

Reply:
I opted for the former strategy, since I previously had trouble getting the latter strategy to work without other build errors.